### PR TITLE
chore: Bump up Trivy to 0.5.4 and upgrade Go mod dependencies

### PR DESCRIPTION
**Pull request by danielpacak**
I also change the behavior in /api/v1/metadata response. Even if getting Trivy version fails, we still return JSON response

### CODE REVIEW

File: pkg/http/api/v1/handler.go [hunk 1]
```diff
@@ -226,7 +216,16 @@ func (h *requestHandler) GetMetadata(res http.ResponseWriter, req *http.Request)
 		"com.github.aquasecurity.trivy.severity":      h.config.Trivy.Severity,
 	}
 
-	if !h.config.Trivy.SkipUpdate {
+	vi, err := h.wrapper.GetVersion()
```
Comment (knqyf263) [inline pkg/http/api/v1/handler.go:219]: info: I feel this change is not `chore` since this affects the behavior of API.

Comment (danielpacak) [reply pkg/http/api/v1/handler.go:219]: That's a good point. Shall we bump adapter up to 0.8.0 then?

Comment (knqyf263) [reply pkg/http/api/v1/handler.go:219]: It's up to you.

Comment (danielpacak) [reply pkg/http/api/v1/handler.go:219]: Okey, pushed change to prepare for 0.8.0 release

**Comment by Mulecharda** [comment]
Thanks for sending!   
  
  
  
  
  
 Mulualem Getachew       
  

  
  
>

**Pull request merged**